### PR TITLE
Add definitions for CTS

### PR DESCRIPTION
**Pull request by lubomir**

### CODE REVIEW

File: fedmsg_meta_umb/cts.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    __description__ = "The Compose Tracking service"
+    __link__ = "https://pagure.io/cts"
+    __docs__ = "https://docs.pagure.org/cts/"
+    __obj__ = "cts"
+
+    def title(self, msg, **config):
+        return msg['topic'].split('.', 2)[-1]
+
+    def subtitle(self, msg, **config):
+        compose = msg['msg']['compose']['compose_info']['payload']['compose']['id']
```
Comment (ralphbean) [inline fedmsg_meta_umb/cts.py:35]: Perhaps wrap this whole section in a `try` block to guard against `KeyError` in case of future message schema changes?

Comment (ralphbean) [reply fedmsg_meta_umb/cts.py:35]: ... if anything goes wrong, just return `None` or `"unexpected message"` and call `log.exception()`

Comment (lubomir) [reply fedmsg_meta_umb/cts.py]: Fair point. The fields under `compose_info` are following schema from productmd, but even that could change eventually.

**Comment by ralphbean** [comment]
LGTM. Thanks!

**Pull request merged**